public/app/types/events.ts

```diff
@@ -125,6 +125,13 @@ export class PanelOptionsChangedEvent extends BusEventBase {
   static type = 'panels-options-changed';
 }
 
+/**
+ * Used by panel panels to tell the otions editor to reload
+ */
+export class PanelOptionsReloadEvent extends BusEventBase {
+  static type = 'panels-options-reload';
```

Comment:
This is required for the canvas panel -- in that panel, we have the notion of a "selected" layer that needs to trigger refreshing what is in the panel options list